docs/guides/src/main/server/logging.adoc

```diff
@@ -7,102 +7,116 @@ title="Configuring logging"
 summary="Learn how to configure Logging"
 includedOptions="log-*">
 
-Keycloak uses the jboss logmanager logging framework. The high-level overview for the available log handlers is shown below:
+Keycloak uses the jboss logmanager logging framework. The following is a high-level overview for the available log handlers:
 
 * root
 ** console (_default_)
 ** file
 
 == Logging: Root configuration
-Logging is done on a per-category basis in Keycloak. You can configure logging for the root log level, or for more specific categories like `org.hibernate` or `org.keycloak`. In this guide, you will learn how to configure logging.
+Logging is done on a per-category basis in Keycloak. You can configure logging for the root log level or for more specific categories such as `org.hibernate` or `org.keycloak`. This guide describes how to configure logging.
 
-=== Root Log level
-The available log levels are listed in the following Table:
+=== Root log levels
```

Comment:
@DGuhr I am wondering if "Root" should be removed  because this section also applies to log levels for category-specific logging unless I am mistaken.

Reply:
Yeh, makes sense. and sorry for the late response :)